### PR TITLE
EZP-31264: Implemented Indexable for ezmatrix FT

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: https://jira.ez.no/browse/EZP-31264

## Description 

Basic impl. `eZ\Publish\SPI\FieldType\Indexable` for the `ezmatrix` field type.

### CODE REVIEW

**Comment by andrerom** [comment]
Given the size, could this go to 1.0 too? :)

**Comment by alongosz** [comment]
Actually, test setup commit(s) need to go into 1.0 anyway.

**Comment by adamwojs** [comment]
Target branch changed to 1.0 as requested by @andrerom / @alongosz 😉

File: composer.json [hunk 1]
```diff
@@ -15,6 +15,13 @@
             "EzSystems\\EzPlatformMatrixFieldtype\\": "src/lib/"
         }
     },
+    "autoload-dev": {
+      "psr-4": {
+        "EzSystems\\EzPlatformMatrixFieldtype\\Integration\\Tests\\": "tests/integration/"
+      }
+    },
+    "minimum-stability": "dev",
```
Comment (mnocon) [inline composer.json]: Shouldn't be needed here (1.0), right? And for master (if it fails to install), we should add the missing packages to require-dev section (and not use minimum-stability, prefer-stable flags).

Comment (adamwojs) [reply composer.json]: Fixed in https://github.com/ezsystems/ezplatform-matrix-fieldtype/pull/25/commits/7dc3dda4c0ded4abf40ae4da7f483b80726b3455

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by adamwojs** [comment]
Done.

**Pull request merged**